src/main/java/org/jenkinsci/plugins/GitLabOAuthUserDetails.java

```diff
@@ -6,21 +6,18 @@
 
 import org.acegisecurity.GrantedAuthority;
 import org.acegisecurity.userdetails.User;
-import org.acegisecurity.userdetails.UserDetails;
+
 
 /**
  * @author Mike
  *
  */
-public class GitLabOAuthUserDetails extends User implements UserDetails {
+public class GitLabOAuthUserDetails extends User {
```

Comment:
This seems like a breaking change.  Why was the `implements UserDetails` removed from the class?

Reply:
Thanks!  I'd never seen that spotbugs warning before.  There appears to be no end to the things that I need to lear!